src/pages/office/office.ts

```diff
@@ -212,6 +224,12 @@ export class OfficePage extends BasePageComponent {
     ];
   }
 
+  ngAfterViewInit(): void {
+    this.dangerousFaultComment.forEach((comment) => {
+      this.inputChangeSubscriptionDispatchingAction(comment, OfficeViewDidAddDangerousFaultComment);
```

Comment:
You will need to store these subscriptions up somewhere and unsubscribe at some point.